### PR TITLE
Mozilla Thunderbird: Added new shortcuts, corrected some wrong shortcuts and improved formatting

### PR DESCRIPTION
**Pull request by mayank7**
Added new shortcuts, corrected shortcut for:
-Undo
-Get new messages for current account
-Get new messages for all account 
and improved formatting by removing extra space and unused lines in Mozilla Thunderbird cheat sheet.

IA Page: https://duck.co/ia/view/mozilla_thunderbird_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Mozilla Thunderbird Cheat Sheet](https://duck.co/ia/view/mozilla_thunderbird_cheat_sheet)

**Description**: Keyboard shortcuts, mouse shortcuts, tips and tricks, ...

**Example Query**: [Mozilla Thunderbird Cheat Sheet](https://beta.duckduckgo.com/?q=Mozilla Thunderbird Cheat Sheet), [Thunderbird Cheat Sheet](https://beta.duckduckgo.com/?q=Thunderbird Cheat Sheet)

**Tab Name**: Answer

**Source**: http://lesliefranke.com/files/reference/thunderbirdcheatsheet.html

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/mozilla_thunderbird_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/mozilla_thunderbird_cheat_sheet)*

File: share/goodie/cheat_sheets/json/mozilla-thunderbird.json [hunk 1]
```diff
@@ -30,43 +25,43 @@
                 "val": "Caret Browsing"
             },
             {
-                "key": "[Ctrl]  [Q]",
+                "key": "[Ctrl] [Q]",
                 "val": "Quit"
             },
             {
-                "key": "[Ctrl]  [T]",
+                "key": "[F5]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mozilla-thunderbird.json]: This shouldn't have `[]` around it!

**Comment by pnpninja** [comment]
@moollaza : just realised i had made a few mistakes.. @mayank7 : thanks for fixing

File: share/goodie/cheat_sheets/json/mozilla-thunderbird.json (hunk 1 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mozilla-thunderbird.json]: @mayank7 Please fix

File: share/goodie/cheat_sheets/json/mozilla-thunderbird.json [hunk 2]
```diff
@@ -22,51 +18,50 @@
...
                 "val": "Send and Receive All Messages"
             },
             {
-                "key": "[Ctrl]  [⏎]",
+                "key": "[Ctrl] [⏎]",
                 "val": "Send Message Now"
             },
             {
-                "key": "[Ctrl]  [⇧]  [⏎]",
+                "key": "[Ctrl] [⇧] [⏎]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mozilla-thunderbird.json]: I think we should replace `[⇧]` with `[Shift]` and `[⏎]` with `[Enter]` for the sake of consistency. We don't use those symbols elsewhere

**Pull request merged**